examples/data-science/pandas/CDC_NDI_Mortality_Syft_Data_Scientist.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<p>Q: The bellow cell looks the same as this one. Is it ok?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Different <code>UCOD_LEADING</code>  values</p>